### ISSUE TITLE
508 Issues from pa11y-rails

### ISSUE THREAD

**Issue by nickbristow** [18F/cg-deck#163]
Results can be recreated using [HTML_CodeSniffer](http://squizlabs.github.io/HTML_CodeSniffer/)

## Results for https://console.cloud.gov

* __Error__ This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.09:1. Recommendation: change background to #1b809e.
 * WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
 * `html > body > div:nth-child(2) > div > div:nth-child(2) > div > div > div:nth-child(4) > h3 > span`
 * `<span class="label label-info">Alpha</span>`

* __Error__ This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 3.93:1. Recommendation: change background to #fefefe.
 * WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
 * `html > body > div:nth-child(2) > div > div:nth-child(2) > div > div > div:nth-child(4) > a > span`
 * `<span class="glyphicon glyphicon-link" aria-hidden="true">Contribute</span>`

* __Error__ The html element should have a lang or xml:lang attribute which describes the language of the document.
 * WCAG2AA.Principle3.Guideline3_1.3_1_1.H57.2
 * `html`
 * `<html ng-app="cfdeck" class="ng-scope"><head><style type="text/css">@c...</html>`

* __Error__ This element has insufficient contrast at this conformance level. Expected a contrast ratio of at least 4.5:1, but text in this element has a contrast ratio of 2.09:1. Recommendation: change background to #1b809e.
 * WCAG2AA.Principle1.Guideline1_4.1_4_3.G18.Fail
 * `html > body > div:nth-child(1) > nav > div > div:nth-child(1) > a > span`
 * `<span class="label label-info">Alpha</span>`

### Summary

__Errors:__ 4

**Comment by msecret** [comment]
This was re-run with the new dashboard and separate issues logged.
